modules/nf-core/sequali/main.nf

```diff
@@ -0,0 +1,58 @@
+process SEQUALI {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'oras://community.wave.seqera.io/library/sequali:0.12.0--c288fa2befb47d0f':
+        'community.wave.seqera.io/library/sequali:0.12.0--07485bec824d914a' }"
+
+    input:
+
+    tuple val(meta), path(reads)
```

Comment:
Does the module use at some point the index file ?

Reply:
No it does not use index files.